packages/theme/src/breadcrumb/responsive.less

```
@@ -0,0 +1,14 @@
+@import '../mixins/common.less';
+@import '../custom.less';
+@import './vars.less';
+
+@breadcrumb-prefix-cls: ~'@{css-prefix}breadcrumb';
+
+.@{breadcrumb-prefix-cls} {
```

Stylelint (17.3.0)

[error] Unknown word breadcrumb-prefix-cls (CssSyntaxError)

(CssSyntaxError)